page-special-meetings.php

```diff
@@ -0,0 +1,18 @@
+<?php get_header(); the_post();?>
+<div class="container">
+	<h1 class="page-title"><?php the_title(); ?></h1>
+	<div class="row page-content" id="<?php echo $post->post_name; ?>">
+		<div class="col-md-9">
+			<?php the_content();?>
+			<?php 
+                $none_term = get_term_by( 'name', 'None', 'people_group' ); 
```

Comment:
Consider adding a check that ensures $none_term is valid (an actual WP_Term object) before passing it to get_special_meetings_by_year_committee(), or add a validity check within get_special_meetings_by_year_committee().  Otherwise we may run into warnings on environments that don't have the 'None' term defined (e.g. on this line, where we access the term's 'term_id' property https://github.com/UCF/BOT-Bootstrap/pull/10/files#diff-78cd5aa3783a74555c9938a2a81d01c6R342).